stdpopsim/utils.py

```diff
@@ -148,3 +148,24 @@ def mask_tree_sequence(ts, mask_intervals, exclude):
     else:
         ts = ts.delete_intervals(mask_intervals)
     return ts
+
+
+def append_common_synonyms(chromosomes):
+    """
+    For common chromosome IDs, add their obvious synonyms if they do not exist already.
+    """
+
+    def add_if_unique(chrom, syn):
+        if syn not in chrom.synonyms:
+            chrom.synonyms.append(syn)
+
+    for chrom in chromosomes:
+        # "1" -> "chr1" and "2L" -> "chr2L"
+        if chrom.id[0].isdigit():
+            add_if_unique(chrom, "chr" + chrom.id)
```

Comment:
Do we need the reverse as well? I don't know if we'll get "chr"-prefixed names by default --- maybe from the ncbi genomes?
```python
        if chrom.id.startswith("chr"):
            add_if_unique(chrom, chrom.id[len("chr"):])
```

Reply:
Hmm, I'm not sure about ncbi genomes. It seems like all the ensembl genomes have chromosome names without the chr prefix, at least from what I've seen and in a few other test cases I've pulled down. If that does come up at some point, we can update this function then to handle those cases?